Mathlib/Analysis/SpecialFunctions/ExpDeriv.lean

```diff
@@ -73,17 +73,13 @@ theorem contDiff_exp : ∀ {n}, ContDiff 𝕜 n exp := by
 #align complex.cont_diff_exp Complex.contDiff_exp
 
 theorem hasStrictDerivAt_exp (x : ℂ) : HasStrictDerivAt exp (exp x) x :=
-  contDiff_exp.ContDiffAt.has_strict_deriv_at' (hasDerivAt_exp x) le_rfl
+  contDiff_exp.contDiffAt.hasStrictDerivAt' (hasDerivAt_exp x) le_rfl
 #align complex.has_strict_deriv_at_exp Complex.hasStrictDerivAt_exp
 
 theorem hasStrictFDerivAt_exp_real (x : ℂ) : HasStrictFDerivAt exp (exp x • (1 : ℂ →L[ℝ] ℂ)) x :=
   (hasStrictDerivAt_exp x).complexToReal_fderiv
 #align complex.has_strict_fderiv_at_exp_real Complex.hasStrictFDerivAt_exp_real
 
-theorem isOpenMap_exp : IsOpenMap exp :=
-  open_map_of_strict_deriv hasStrictDerivAt_exp exp_ne_zero
-#align complex.is_open_map_exp Complex.isOpenMap_exp
```

Comment:
I see that you remove an import and a lemma. What's the plan here: move the lemma somewhere else (then you should open a `mathlib3` PR doing the same)? Restore it once the inverse function theorem is ported?

Reply:
I think that's forward porting leanprover-community/mathlib#19139

Reply:
I will rebase on top of a fresh `mathport` output in ~15min.

Reply:
It didn't go through `mathport` yet. Should we manually update SHA sum in the header?

Reply:
It's gone through mathport now